<ui version="4.0" >
 <class>AppAccessPointConfigPage</class>
 <widget class="QWidget" name="AppAccessPointConfigPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Application Access</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="label" >
     <property name="font" >
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Inter-Process Communication Configuration</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Access Point Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="txtAccessPointName" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Enable Access</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QCheckBox" name="chkEnableAccess" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Access Point Address</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="txtAddress" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Access Point Port</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLineEdit" name="txtPort" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Applications connected to me</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QPushButton" name="cmdAppsToMeRemove" >
        <property name="text" >
         <string>&amp;Remove</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>131</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QListWidget" name="lstAppsToMe" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Applications I am connected to</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QPushButton" name="cmdMeToAppsAdd" >
        <property name="text" >
         <string>&amp;Add</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="cmdMeToAppsRemove" >
        <property name="text" >
         <string>&amp;Remove</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="3" >
       <widget class="QListWidget" name="lstMeToApps" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
